packages/eslint-config-react-app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "eslint-config-react-app",
+  "version": "0.4.3",
+  "description": "ESLint configuration used by Create React App",
+  "repository": "facebookincubator/create-react-app",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/facebookincubator/create-react-app/issues"
+  },
+  "files": [
+    "index.js"
+  ],
+  "peerDependencies": {
+    "babel-eslint": "^6.1.2",
+    "eslint": "^3.5.0",
+    "eslint-plugin-flowtype": "^2.18.1",
+    "eslint-plugin-import": "^1.12.0",
+    "eslint-plugin-jsx-a11y": "^2.2.2",
+    "eslint-plugin-react": "^5.2.2"
```

Comment:
While we're at it, can you look into updating us to `eslint-plugin-react@6`? Have any rules changed?